src/sentry/notifications/notifications/rules.py

```diff
@@ -101,6 +102,8 @@ def get_context(self) -> MutableMapping[str, Any]:
             "slack_link": get_integration_link(self.organization, "slack"),
             "has_alert_integration": has_alert_integration(self.project),
             "issue_type": GROUP_TYPE_TO_TEXT.get(self.group.issue_type, "Issue"),
+            "subtitle": self.event.title,
+            "default_issue_data": [("Issue Data", get_default_data(self.event)), None],
```

Comment:
This can be called something else if someone can think of something better that's generic.